CONTRIBUTING.md

```diff
@@ -99,7 +99,7 @@ Example:
 
 ### Reporting upstream browser bugs
 
-Sometimes bugs reported to us are actually caused by bugs in the browser(s) themselves, not bugs in Bootstrap per se.
+Sometimes bugs reported to us are actually caused by bugs in the browser(s) themselves, not bugs in Bootstrap per say.
```

Comment:
The original was correct as-is.
See https://en.wiktionary.org/wiki/per_se#English